test_mvp.ipynb

```diff
@@ -2,56 +2,157 @@
  "cells": [
```

Comment:
<p>Can we add a <code># header</code> here so its clear we are switching subjects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sequgen/notebooks/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>